### ISSUE TITLE
executehostcommand / reloading config deletes top line instead of adding it to scrollback

### ISSUE THREAD

**Issue by DejayRezme** [nushell/nushell#5305]
### Describe the bug

I assume the problem is actually with 'executehostcommand'. When I press the hotkey reload the config file the current top line gets deleted instead so when you scroll up your terminal the line isn't there.

This is on windows terminal. Not sure if this happens only there but I have other issues with nushell deleting lines too, for example tab completion can delete all lines on the screen.

```
    { # Reload the config file
      name: reload_config
      modifier: control
      keycode: char_w
      mode: emacs
      event: {
        send: executehostcommand,
        cmd: $"source '($nu.config-path)'"
	  }
	}
```

### How to reproduce

1. ls
2. press ctrl-w

### Expected behavior

Lines should not be deleted but scrolled away

### Configuration

| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.61.0                                   |
| branch             |                                          |
| tag                |                                          |
| short_commit       | 3783c19d                                 |
| commit_hash        | 3783c19d02f4a3b2f35cd26b79daabcb70b04880 |
| commit_date        | 2022-04-12 20:38:15 +00:00               |
| build_os           | windows-x86_64                           |
| rust_version       | rustc 1.60.0 (7737e0b5c 2022-04-04)      |
| rust_channel       | stable-x86_64-pc-windows-msvc            |
| cargo_version      | cargo 1.60.0 (d1fd9fe2c 2022-03-01)      |
| pkg_version        | 0.61.0                                   |
| build_time         | 2022-04-12 21:07:36 +00:00               |
| build_rust_channel | release                                  |
| features           | dataframe, default, trash, which, zip    |
| installed_plugins  |                                          |

**Comment by sholderbach** [comment]
Mhh that sounds similar to the other problem we have with prompt initialization on Windows eating a previous output line nushell/reedline#398

**Comment by DejayRezme** [comment]
Yeah I have this issue in different circumstances like with the tab completion menu , this (reload config eats line) was just the easiest to reproduce.

Reading a bit in 398 I can reproduce the same issue  with wezterm and alacritty on windows. So either the default linux terminal does something different or it's a difference how operating systems are handled (something like /r vs /n?)
